deso/block.go

```diff
@@ -509,14 +532,9 @@ func (node *Node) getBuyNowNFTBidOps(txn *lib.MsgDeSoTxn, utxoOpsForTxn []*lib.U
 		return nil
 	}
 
-	// We only care about NFT bids that generate creator royalties. This only occurs for NFT bids on Buy Now NFTs.
-	// Only NFT bids on Buy Now NFTs will have NFTBidcreatorRoyaltyNanos > 0. Even in the case that an NFT is a buy now,
-	// it is possible that there are no royalties to the creator coin generated. In this situation, it is okay to exit
-	// early since there are not output to add.
-	if nftBidOp.NFTBidCreatorRoyaltyNanos == 0 {
```

Comment:
Why remove this if statement?

Reply:
Because there may be additional royalties from the NFT split. Since there now may be royalties to creator coins other than the creator coin of the poster, we can't exit early by only checking if the royalty to the poster's coin is zero.

Reply:
Gotcha. Should this if statement be added to the `nftBidOp.PrevCoinEntry.CoinsInCirculationNanos >= node.Params.CreatorCoinAutoSellThresholdNanos` check? I am not sure if an operation with `0` nanos is OK or not.

Reply:
I've added the conditional check - lmk if there's anything else concerning in this one